portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase

```diff
@@ -72,6 +72,49 @@ definition {
 		}
 	}
 
+	@description = "LPS-170580 - Verify alert in the middle of stack is closed, the others fill the stack"
+	@priority = "4"
+	test CanTransitionToFillGapOfDeletedToast {
```

Comment:
Given the known first (top) alert position from two stacked toast alert success messages